src/main/java/ProgramB.java

```diff
@@ -0,0 +1,57 @@
+import java.text.DecimalFormat;
+import java.util.Scanner;
+/**
+ * 1. 기초 알고리즘 b. 두 개의 숫자를 입력받고, n~m 까지의 연속된 수의 합을 구하는 프로그램
+ */
+public class ProgramB {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        String inputN;
+        String inputM;
+        Verification vrfc = new Verification();
+
+        do {
+            System.out.println("정수 값을 입력해주세요.");
+            inputN = scan.next();
+        } while (vrfc.isString(inputN));
+        do {
+            System.out.println("정수 값을 입력해주세요.");
+            inputM = scan.next();
+        } while (vrfc.isString(inputM));
```

Comment:
```suggestion
        inputN = inputPrompt("정수 값을 입력해주세요.");
        inputM = inputPrompt("정수 값을 입력해주세요.");
```
정수가 아닌 입력을 처리하기 위해 만들어진 인풋 코드 같네요~
좋은 시도입니다.

코드의 중복이 있는데, 메서드로 한번 더 분리를 한다면 가독성 좋은 코드로 바뀔 수 있어보여요.
위 코드처럼 작성하면 동일하게 동작하도록 메서드를 한번 더 분리해보세요.

Reply:
이 부분도 줄일 수 있겠다는 생각을 했었는데 시간에 쫓겨서 넘어갔습니다. 다시 생각해보도록 하겠습니다.